status/status.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // NamePrefix is a common name prefix for this service.
-const NamePrefix = "HACBS Test"
+const NamePrefix = "AppStudio Test"
```

Comment:
I think that this will need to be the exception to the `AppStudio` usage based on [RHTAP-335](https://issues.redhat.com/browse/RHTAP-335) guidelines (and the [RHTAP-511](https://issues.redhat.com/browse/RHTAP-511) decision) and should be `Red Hat Trusted App`.

Reply:
Looks good, thanks